src/solver/hydro/management/management.h

```diff
@@ -107,39 +106,37 @@ class HydroManagement final
                     unsigned int maxNbYearsInParallel,
                     Solver::IResultWriter& resultWriter);
 
-    ~HydroManagement();
-
-    double randomReservoirLevel(double min, double avg, double max);
+    static double randomReservoirLevel(double min, double avg, double max, MersenneTwister& random);
 
     //! Perform the hydro ventilation
     void makeVentilation(double* randomReservoirLevel,
-                        Solver::Variable::State& state,
-                        uint y,
-                        uint numSpace);
+                         Solver::Variable::State& state,
+                         uint y,
+                         uint numSpace);
 
-    ALL_HYDRO_VENTILATION_RESULTS& ventilationResults() { return ventilationResults_; }
+    HYDRO_VENTILATION_RESULTS& ventilationResults() { return ventilationResults_; }
```

Comment:
While were are here: this should be `const HYDRO_VENTILATION_RESULTS& ventilationResults() const`
to make clear that the result will not be changed outside.